modules/tinymce/src/plugins/link/main/ts/core/Utils.ts

```diff
@@ -133,16 +133,16 @@ const handleExternalTargets = (href: string, assumeExternalTargets: AssumeExtern
 
 const applyLinkOverrides = (editor: Editor, linkAttrs: LinkAttrs): LinkAttrs => {
   const newLinkAttrs = { ...linkAttrs };
-  if (!(Settings.getRelList(editor).length > 0) && Settings.allowUnsafeLinkTarget(editor) === false) {
+  if (Options.getRelList(editor).length === 0 && !Options.allowUnsafeLinkTarget(editor)) {
```

Comment:
Should this be `Options.getRelList(editor).length === 0 || Options.allowUnsafeLinkTarget(editor)` to be the same thing as before seems partially flipped?

Reply:
No that'd change it... it did `!(Settings.getRelList(editor).length > 0)` which means `isEmpty` hence the `Options.getRelList(editor).length === 0` part. Then `allowUnsafeLinkTarget` is always going to be a boolean now so doing `Settings.allowUnsafeLinkTarget(editor) === false` is the same as `!Options.allowUnsafeLinkTarget(editor)`.

Reply:
What you said would have been correct if the end parenthesis included the whole statement but it was only around the `> 0` check.

Reply:
Right, I missed that the parentheses was in the middle there.